packages/simcore-sdk/tests/integration/test_node_ports_aws_s3_cli.py

```diff
@@ -0,0 +1,417 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import asyncio
+import filecmp
+import os
+import urllib.parse
+from collections.abc import AsyncIterator, Callable
+from pathlib import Path
+from typing import Final
+from unittest.mock import AsyncMock
+from uuid import uuid4
+
+import aioboto3
+import aiofiles
+import pytest
+from faker import Faker
+from models_library.progress_bar import ProgressReport
+from pydantic import AnyUrl, ByteSize, parse_obj_as
+from servicelib.file_utils import remove_directory
+from servicelib.progress_bar import ProgressBarData
+from servicelib.utils import logged_gather
+from settings_library.aws_s3_cli import AwsS3CliSettings
+from simcore_sdk.node_ports_common import aws_s3_cli
+
+pytest_simcore_core_services_selection = [
+    "migration",
+    "postgres",
+    "storage",
+    "redis",
+]
+
+pytest_simcore_ops_services_selection = [
+    "minio",
+    "adminer",
+]
+
+
+WAIT_FOR_S3_BACKEND_TO_UPDATE: Final[float] = 1.0
+
+
+@pytest.fixture
+async def cleanup_bucket_after_test(aws_s3_cli_settings: AwsS3CliSettings) -> None:
```

Comment:
we already have fixtures for this. Could we maybe re-use these instead? we can go through them if needed.

Reply:
As discussed for now test are 1-2-1 with RClone tests.